files/default/neutron-ha-tool.py

```diff
@@ -561,13 +563,14 @@ def replicate_dhcp(qclient, noop=False):
 
 
 def migrate_l3_routers_from_agent(qclient, agent, targets,
-                                  noop, wait_for_router, delete_namespace):
+                                  noop, wait_for_router, delete_namespace,
+                                  agent_picker):
```

Comment:
@vuntz what would you recommend? My logic here was to add the new argument as the next arg. So if you have kwargs:

```python
def fun1(a, b, c=1, d=2):
```

becomes

```python
def fun1(a, b, newarg, c=1, d=2):
```

and if you only have args:

```python
def fun3(a, b, c):
```

becomes

```python
def fun3(a, b, c, newarg):
```